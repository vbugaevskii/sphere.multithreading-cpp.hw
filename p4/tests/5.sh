sleep 0.1 && false || echo running in background | cat &
echo pre
sleep 0.2
echo post

echo exit 71 |  bash &
sleep 0.1
