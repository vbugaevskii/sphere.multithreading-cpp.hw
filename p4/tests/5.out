pre
running in background
post
